experimenter/experimenter/nimbus_ui_new/templates/nimbus_experiments/detail.html

```diff
@@ -1,14 +1,365 @@
+              <td class="{% if experiment.risk_revenue %}text-danger font-weight-bold{% endif %}">
+                {% if experiment.risk_revenue != None %}
+                  {{ experiment.risk_revenue|yesno }}
+                {% else %}
+                  Not Set
+                {% endif %}
+              </td>
+            </tr>
+            <tr>
+              <th colspan="3">{{ RISK_QUESTIONS.PARTNER }}</th>
+              <td class="{% if experiment.risk_partner_related %}text-danger font-weight-bold{% endif %}">
+                {% if experiment.risk_partner_related != None %}
+                  {{ experiment.risk_partner_related|yesno }}
+                {% else %}
+                  Not Set
+                {% endif %}
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
+    <!-- Audience Card -->
+    <div class="card mb-3">
+      <div class="card-header">
+        <h4>Audience</h4>
+      </div>
+      <div class="card-body">
+        <table class="table table-striped">
+          <tbody>
+            <tr>
+              <th>Channel</th>
+              <td>{{ experiment.channel.title|default:"No Channel" }}</td>
+              <th>Advanced Targeting</th>
+              <td>{{ experiment.targeting_config.name }}</td>
+            </tr>
+            <tr>
+              <th>Minimum version</th>
+              <td>{{ experiment.firefox_min_version|parse_version }}</td>
+              <th>Maximum version</th>
+              <td>{{ experiment.firefox_max_version|parse_version }}</td>
+            </tr>
+            <tr>
+              <th>Locales</th>
+              <td>
+                {{ experiment.locales.all|join:"<br>"|default:"All Locales" }}
+              </td>
+              <th>Countries</th>
+              <td>
+                {{ experiment.countries.all|join:"<br>"| default:"All Countries" }}
+              </td>
+            </tr>
+            <tr>
+              <th>Expected enrolled clients</th>
+              <td>{{ experiment.total_enrolled_clients }}</td>
+              <th>Population %</th>
+              <td>{{ experiment.population_percent|floatformat:"-1" }}%</td>
+            </tr>
+            <tr>
+              <th>Sticky enrollment</th>
+              <td colspan="3">{{ experiment.is_sticky }}</td>
+            </tr>
+            <tr>
+              <th>Required experiments</th>
+              <td>
+                {{ experiment.required_experiments.all|join:"<br>"|default:"None" }}
+              </td>
+              <th>Excluded experiments</th>
+              <td>
+                {{ experiment.excluded_experiments.all|join:"<br>"|default:"None" }}
+              </td>
+            </tr>
+            <tr>
+              <th>Full targeting expression</th>
+              <td colspan="3">{{ experiment.targeting }}</td>
+            </tr>
+            <tr>
+              <th>Recipe JSON</th>
+              <td colspan="3">{{ experiment.targeting }}</td>
```

Comment:
Oops this should be recipe json not targeting 🙈

Reply:
yup It was intentional, we need to build recipe JSON, filed a separate ticket for that